DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -303,6 +315,10 @@ static function StartMapSpecificFlags(FlagBase flagbase, string start_map)
             flagbase.SetBool('KnowsSmugglerPassword',true,,-1);
             flagbase.SetBool('MetSmuggler',true,,-1);
             break;
+        case "10_Paris_Chateau":
```

Comment:
I feel like these could instead be done with cascading cases on the int start_map_val, like

```
case 109:
    flagbase.SetBool('SandraWentToCalifornia',true,,-1);
case 119:
    flagbase.SetBool('MeetTobyAtanwe_played',true,,-1);
    flagbase.SetBool('FemJCMeetTobyAtanwe_played',true,,-1);
```

or maybe backwards with later starts at the top of the switch/case and falling through to the earlier starts?